frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card.tsx

```diff
@@ -41,15 +41,20 @@ const storageClassResource: FirehoseResource = {
   prop: 'storageclass',
 };
 
-const getNoobaaStorageClasses = (storageClassesList) =>
-  storageClassesList.filter((sc) => _.endsWith(_.get(sc, 'provisioner'), 'noobaa.io/obc'));
+const getNoobaaStorageClassList = (storageClassesList: K8sResourceKind[]): K8sResourceKind[] =>
+  storageClassesList.filter((sc: K8sResourceKind) =>
+    _.endsWith(_.get(sc, 'provisioner'), 'noobaa.io/obc'),
+  );
 
-const getNoobaaObcCount = (obcList, noobaaStorageClasses) => {
-  const result = obcList.filter((o) => {
-    const storageClassName = _.get(o, 'spec.storageClassName');
-    return noobaaStorageClasses.some((sc) => getName(sc) === storageClassName);
+const getNoobaaObcList = (
+  obcList: K8sResourceKind[],
+  noobaaStorageClassList: K8sResourceKind[],
+): K8sResourceKind[] => {
```

Comment:
```suggestion
): K8sResourceKind[] => 
```

Reply:
I found that version more readable hence changed on purpose.

Reply:
I think this would be more concise. How is it more readable? Looking at the code It doesn't look like block statements would make it any more readable.

Reply:
It was getting cluttered and having two loops hence I used. I will change since its waste of time to argue on petty things that do not boost productivity.
Thanks!